test_main.py

```diff
@@ -35,3 +35,23 @@ def test_extract_text_request_exception():
         response = client.get("/extract-text", params={"url": url})
         assert response.status_code == 400
         assert response.json() == {"detail": "Request failed"}
+
+
+def test_extract_text_bermuda():
+    url = "https://thinkingofbermuda.com"
+    html_content = "<html><body><p>Welcome to Bermuda!</p></body></html>"
+    expected_text = "Welcome to Bermuda!"
+
+    headers = {
+        "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3",
+        "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8",
+    }
+
+    with patch("requests.get") as mock_get:
```

Comment:
There is no point in mocking this. Just make sure that status code = 200

Reply:
Thank you for the feedback. I understand your point. I'll update the test to directly check the status code without mocking the request.

### Updated `test_main.py`

```python
def test_extract_text_bermuda():
    url = "https://thinkingofbermuda.com"
    
    response = client.get("/extract-text", params={"url": url})
    assert response.status_code == 200
    assert "text" in response.json()
```

This test will now directly check the status code and ensure that the response contains the "text" key, without mocking the request.

I'll make this change and update the PR.